modules/product/src/main/java/com/opengamma/strata/product/swap/FixedAccrualMethod.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2018 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product.swap;
+
+import org.joda.convert.FromString;
+import org.joda.convert.ToString;
+
+import com.opengamma.strata.collect.named.EnumNames;
+import com.opengamma.strata.collect.named.NamedEnum;
+
+/**
+ * The method of accruing interest based on the fixed rate of a fixed swap leg.
+ * <p>
+ * Two methods of accrual are supported - simple and compounded.
+ */
+public enum FixedAccrualMethod implements NamedEnum {
+
+  /**
+   * The simple method.
+   * <p>
+   * Simple interest is accrued. This is determined by multiplying the fixed rate by the notional and the accrual factor.
+   * <></>
+   * This is the most common type for vanilla swaps.
+   */
+  SIMPLE,
+
+  /**
+   * The BRL style compounded method.
+   * <p>
+   * OpenGamma, "Brazilian Swaps" (2013).
```

Comment:
Although we include references, we try to make them additional, so this should include a short description  of the method